Code/src/main/java/nl/utwente/group10/haskell/type/Type.java

```diff
@@ -30,7 +30,15 @@
     /**
      * @return The Haskell (type) representation of this type.
      */
-    public abstract String toHaskellType();
+    public String toHaskellType() {
+    	return this.toHaskellType(0);
+    }
+
+    /**
+     * @param The fixity of the context the type is shown in.
```

Comment:
Could perhaps use a short definition, as well as a reference of what the values mean (i.e. that they should always be between 0 and 9).